packages/monaco/src/browser/monaco-languages.ts

```diff
@@ -15,19 +15,26 @@
  ********************************************************************************/
 
 import { injectable, inject, decorate } from 'inversify';
-import { MonacoLanguages as BaseMonacoLanguages, ProtocolToMonacoConverter, MonacoToProtocolConverter } from 'monaco-languageclient';
+import {
+    MonacoLanguages as BaseMonacoLanguages, ProtocolToMonacoConverter,
+    MonacoToProtocolConverter, WorkspaceSymbolParams, CancellationToken, SymbolInformation
+} from 'monaco-languageclient';
 import { Languages, Diagnostic, DiagnosticCollection, Language } from '@theia/languages/lib/browser';
 import { ProblemManager } from '@theia/markers/lib/browser/problem/problem-manager';
 import URI from '@theia/core/lib/common/uri';
 import { Mutable } from '@theia/core/lib/common/types';
 import { Disposable } from '@theia/core/lib/common/disposable';
-import { WorkspaceSymbolProvider } from 'monaco-languageclient/lib/services';
 import { MonacoDiagnosticCollection } from 'monaco-languageclient/lib/monaco-diagnostic-collection';
 
 decorate(injectable(), BaseMonacoLanguages);
 decorate(inject(ProtocolToMonacoConverter), BaseMonacoLanguages, 0);
 decorate(inject(MonacoToProtocolConverter), BaseMonacoLanguages, 1);
 
+export interface WorkspaceSymbolProvider {
```

Comment:
How about reference `WorkspaceSymbolProvider` from `language-client-services` instead of redeclaring?